src/NetSparkle/Downloaders/WebRequestAppCastDataDownloader.cs

```diff
@@ -4,12 +4,13 @@
 using System.IO;
 using System.Net;
 using System.Net.Security;
+using System.Runtime.InteropServices;
```

Comment:
Is this necessary?

Reply:
Removed, this was from me trying to fix the crash with mono and verifying certs.